src/tx/builder/constants.ts

```diff
@@ -88,7 +92,9 @@ export const ID_TAG = {
   channel: ID_TAG_CHANNEL
 }
 
-export const PREFIX_ID_TAG = {
+export const PREFIX_ID_TAG: {
+  [prefix: string]: number
```

Comment:
This actually make the type less precise, I proposing to add `as const` at the end instead. Then the strings defined inside would be checkable by type system.